docs/querying/basics.md

```diff
@@ -334,10 +367,10 @@ or aggregated your data sufficiently, switch to graph mode. If the expression
 still takes too long to graph ad-hoc, pre-record it via a [recording
 rule](../configuration/recording_rules.md#recording-rules).
 
-This is especially relevant for Prometheus's query language, where a bare
+This consideration is especially relevant for Prometheus's query language, where a bare
 metric name selector like `api_http_requests_total` could expand to thousands
-of time series with different labels. Also keep in mind that expressions which
-aggregate over many time series will generate load on the server even if the
-output is only a small number of time series. This is similar to how it would
+of time series with different labels. Also, keep in mind that expressions that
+aggregate over many number of time series samples will generate load on the server even if the
+output is only a small number of time series samples. This is similar to how it would
```

Comment:
```suggestion
of time series with different labels. Also, keep in mind that expressions that
aggregate over many time series will generate load on the server even if the
output is only a small number of time series. This is similar to how it would
```